src/content/docs/solvro/Strategy.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: Strategia KN Solvro
+sidebar:
+  order: 1
+description: Długoterminowy plan strategiczny KN Solvro
+---
+
+import {
+  Steps,
+  Card,
+  CardGrid,
+  LinkCard,
+  Aside,
+} from "@astrojs/starlight/components";
+
+> <b>
+>   <i>"Otwieramy drzwi do świata IT" ~ motto KN Solvro</i>
+> </b>
+
+## Początki
+
+W 2018 roku grupa studentów napędzona kreatywnością i chęcią tworzenia założyła **Koło Naukowe Solvro**. Od tamtego momentu, przez te lata nasza organizacja podejmowała coraz to nowsze inicjatywy. Zarządy zmieniały się, a wraz z czasem początkowa wizja koła zanikała.
+
+W 2024 roku, po objęciu władzy przez VII Zarząd, przeprowadzono restrukturyzację. Pół roku później, podczas wyjazdu do Lubawki, powstał zarys działań. Pod koniec roku opracowano dokument LTSP (Long Term Strategic Plan), który wytycza ścieżkę dla przyszłych zarządów, przekładając wiedzę obecnych członków na wartościowe wnioski zawarte w planie.
+
+**KN Solvro** ma przed sobą świetlaną przyszłość, jednak to przyszli członkowie i zarządy będą odpowiadać za jej pielęgnowanie oraz dalszy rozwój 🚀.
+
+## Nasze fundamenty
+
+Wywodząc się z Politechniki Wrocławskiej, podążamy jej kierunkiem, jednocześnie definiując własną misję i wizję.
+
+### Misja
+
+> <b>
+>   „Jesteśmy mostem między teorią i praktyką, kształcąc kolejne pokolenia
+>   studentów napędzających innowacje na Politechnice Wrocławskiej.”
+> </b>
+
+### Wizja
+
+> <b>
+>   „Wspieramy społeczność akademicką poprzez technologię, współpracę i wiedzę.”
+> </b>
```

Comment:
https://www.markdownguide.org/basic-syntax/#emphasis

jak możesz to używaj markdowna